src/rx-query.ts

```diff
@@ -259,8 +254,13 @@ export class RxQueryBase<
         this._execOverDatabaseCount = this._execOverDatabaseCount + 1;
         this._lastExecStart = now();
 
-
         if (this.op === 'count') {
+            // if we have a persisted query cache result, use the result
+            if (this._persistentQueryCacheResult) {
+                // TODO: correct this number, but how?
```

Comment:
so what happens now with counts? are they permanently wrong until the persistent query cache is busted?

Reply:
I currently persist the result of a `count` query every time there is a change handled by event-reduce (and not actually querying). I couldn't produce a mistake, but there is no correction mechanism either. Feels scary, have no solution to offer (except storing all ID's of the count query and doing the same we do with regular queries, but that sounds like adding lots of slowness).

Reply:
mmm, so error(persistent count) == error(event-reduce). i can live with that, event-reduce does make mistakes but they tend to be rare/edge-casey.

Reply:
event-reduce shouldn't make mistakes after we upgrade to his newest version:

https://github.com/pubkey/event-reduce/issues/444#issuecomment-1829793615

(which i'm doing today)